### PR TITLE
Cge 19 registration

### PR DESCRIPTION
**Pull request by Wolshebnik**

### CODE REVIEW

File: src/main/react/webpack_registration/web_pack/components/pages/FormRegister/Form.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    setErrorsInput({});
+  }, [isLogin, setDataInputsState]);
+
+  useEffect(() => {
+    if (0 !== Object.keys(errorsInput).length) {
+      setReadyToEnter(false);
+      return;
+    }
+    const check = false;
+    if (
```
Comment (rpkyrych) [inline src/main/react/webpack_registration/web_pack/components/pages/FormRegister/Form.js]: You should fix formatting here. If conditions if so long - create new function for checking them all.

File: src/main/react/webpack_registration/web_pack/components/pages/FormRegister/Form.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+      (0 === Object.keys(errorsInput).length &&
+        isChecking &&
+        checkAllInputs(check)) ||
+      isSubmitting
+    ) {
+      setReadyToEnter(true);
+    }
+    if (
```
Comment (rpkyrych) [inline src/main/react/webpack_registration/web_pack/components/pages/FormRegister/Form.js]: The same about formatting.

File: src/main/react/webpack_registration/web_pack/components/pages/FormRegister/extra/Extra.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            event.target.checked ? setIsChecking(true) : setIsChecking(false);
+          }}
+        />
+        <label className="check" htmlFor="check">
+          <svg width="10" height="8" viewBox="0 0 10 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <mask id="path-1-inside-1" fill="white">
+              <path fillRule="evenodd" clipRule="evenodd" d="M10.0007 0.835938L4.10237 7.00116L0.998047 3.75638"/>
+            </mask>
+            <path fillRule="evenodd" clipRule="evenodd" d="M10.0007 0.835938L4.10237 7.00116L0.998047 3.75638" fill="black" fillOpacity="0.01"/>
+            <path d="M4.10237 7.00116L2.65722 8.38375L4.10238 9.89429L5.54752 8.38375L4.10237 7.00116ZM8.55551 -0.546645L2.65722 5.61858L5.54752 8.38375L11.4458 2.21852L8.55551 -0.546645ZM5.54751 5.61857L2.44319 2.37379L-0.447099 5.13897L2.65722 8.38375L5.54751 5.61857Z" fill="white" mask="url(#path-1-inside-1)"/>
```
Comment (rpkyrych) [inline src/main/react/webpack_registration/web_pack/components/pages/FormRegister/extra/Extra.js]: Is it possible to move all these parameters to .css file?

File: src/main/react/webpack_registration/web_pack/components/pages/FormRegister/inputBase.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    text: "Введите ваш E-mail или логин",
+    type: "text",
+    dataError: "Укажите действующий адрес электронной почты",
+    message: "Please enter valid email/login or password"
+  },
+  {
+    id: "password",
+    text: "Введите ваш пароль",
+    type: "password",
+    dataError: "Не верный пароль"
```
Comment (rpkyrych) [inline src/main/react/webpack_registration/web_pack/components/pages/FormRegister/inputBase.js]: You should update this message to "Неправильный пароль".

File: src/main/react/webpack_registration/web_pack/components/pages/FormRegister/inputBase.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  }
+];
+
+const errorsServer = [
+  {
+    errorStatus: "Введите верный логин или email.",
+    message: "Please enter valid email/login or password"
+  },
+  {
+    errorStatus: "Этот email уже существует.",
```
Comment (rpkyrych) [inline src/main/react/webpack_registration/web_pack/components/pages/FormRegister/inputBase.js]: You should update the message to "Этот email уже зарегистрирован".

File: src/main/react/webpack_registration/web_pack/components/pages/hooks/doFetch.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+export default whatPage => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [response, setResponse] = useState(null);
+  const [errorsResponse, setError] = useState(null);
+  const [data, setData] = useState({});
+
+  let baseUrl = `http://localhost:8080/auth/${whatPage}`;
```
Comment (rpkyrych) [inline src/main/react/webpack_registration/web_pack/components/pages/hooks/doFetch.js]: What will happen if you here just put "/auth/${whatPage}" ?

File: src/main/react/webpack_registration/webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+      warnings: true,
+      errors: true
+    },
+    port: 3999
+  },
+  // resolve: {
+  //   extensions: [".js", ".css", ".scss", "jpeg", "jpg", "png", ".svg"],
+  //   alias: {
+  //     "@modules": path.resolve(__dirname, "web_pack/components"),
+  //     "@": path.resolve(__dirname, "wep_pack")
```
Comment (rpkyrych) [inline src/main/react/webpack_registration/webpack.config.js]: Why do you keep this commented code?

File: src/main/react/webpack_registration/webpack.config.js [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+          }
+        ]
+      },
+
+      // {
+      //   test: /\.js$/,
+      //   exclude: /node_modules/,
+      //   loader: {
+      //     loader: "babel-loader",
+      //     options: babelOptions()
```
Comment (rpkyrych) [inline src/main/react/webpack_registration/webpack.config.js]: The same about commented code...

File: src/main/react/webpack_registration/webpack.config.js [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+    new HtmlWebpackPlugin({
+      template: "html/registration.html",
+      filename: "html/registration.html",
+      // minify: {
+      //   collapseWhitespace: isProd
+      // }
+    }),
+    // new CleanWebpackPlugin(),
+    // new CopyWebpackPlugin([
+    //   {
```
Comment (rpkyrych) [inline src/main/react/webpack_registration/webpack.config.js]: The same...

**Comment by Wolshebnik** [comment]
not relevant

**Pull request closed without merging**